tests/catch/unit/stream/hipStreamValue.cc

```diff
@@ -352,93 +471,86 @@ DEFINE_STREAM_WAIT_VAL_TEST_CASES_INT64("NoMask_And",
                                                     0x0F0F0F0F0F0F0F0F, 0X1F0F0F0F0F0F0F0F))
 DEFINE_STREAM_WAIT_VAL_TEST_CASES_INT64("NoMask_Nor",
                                         TEST_WAIT64(hipStreamWaitValueNor, 0x4724724747247247,
-                                                    static_cast<int64_t>(0xbddbddbdbddbddbd),
-                                                    static_cast<int64_t>(0xbddbddbdbddbddb3)))
+                                                    0xbddbddbdbddbddbd, 0xbddbddbdbddbddb3))
 #undef DEFINE_STREAM_WAIT_VAL_TEST_CASES_INT64
 
-#endif
+#define DEFINE_HIP_STREAM_VALUE(TYPE, BITS, ...) hipStream##TYPE##Value##BITS(__VA_ARGS__)
```

Comment:
This macros are not used anywhere anymore:

DEFINE_HIP_STREAM_VALUE
NEG_TEST_ERROR_CHECK